src/bin/vip-config-software.js

```diff
@@ -2,10 +2,36 @@
 
 import command from '../lib/cli/command';
 
+const usage = 'vip config software';
+const exampleUsage = 'vip @example-app.develop config software';
+const exampleUsageNode = 'vip @example-node-app.develop config software';
+
+// Command examples
+const examples = [
+	{
+		usage: `${ exampleUsage } get`,
+		description: 'Retrieve the current software configuration for all supported components.',
+	},
+	{
+		usage: `${ exampleUsage } get wordpress --include available_versions`,
+		description:
+			'Retrieve the current software configuration for WordPress including available versions.',
+	},
+	{
+		usage: `${ exampleUsage } update wordpress 6.4`,
+		description: 'Update WordPress application environment to 6.4.x.',
```

Comment:
Suggested:
`Update the version of WordPress on a WordPress environment to 6.4.x.`